WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,222 @@
+
+                        Button(Localization.addParameterButton) {
+                            // todo
+                        }
+                        .buttonStyle(PlusButtonStyle())
+                        .padding([.leading, .trailing], Layout.contentSpacing)
+                        .padding(.bottom, Layout.parametersVerticalSpacing)
+                    }
+                    .background(Color(.systemBackground))
+                }
+
+                // Remaining characters and final destination
+                VStack(alignment: .leading) {
+                    Text(viewModel.remainingCharactersLabel)
+                        .subheadlineStyle()
+                        .padding(.bottom, Layout.contentVerticalSpacing)
+
+                    Text(viewModel.finalDestinationLabel)
+                        .subheadlineStyle()
+                }
+                .padding([.leading, .trailing], Layout.contentSpacing)
+                .padding([.top, .bottom], Layout.contentVerticalSpacing)
+                .background(Color(.systemGray6))
+
+                Spacer()
+            }
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .background(Color(.systemGray6))
+            .navigationBarTitleDisplayMode(.inline)
+            .navigationTitle(Localization.adDestination)
+            .toolbar {
+                ToolbarItem(placement: .cancellationAction) {
+                    Button(Localization.cancel) {
+                        dismiss()
+                    }
+                }
+            }
+        }
+    }
+    private func sectionHeading(title: String) -> some View {
+        Text(title.uppercased())
+            .subheadlineStyle()
+            .frame(maxWidth: .infinity, alignment: .leading)
+            .padding(Layout.sectionHeadingPadding)
+    }
+
+    private func destinationItem(title: String,
+                                 subtitle: String,
+                                 type: DestinationType,
+                                 showBottomDivider: Bool = false) -> some View {
+        HStack(alignment: .center) {
+            if type == viewModel.selectedDestinationType {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .foregroundColor(Color(uiColor: .accent))
+            } else {
+                Image(systemName: "checkmark")
+                    .padding(.leading, Layout.contentSpacing)
+                    .padding(.trailing, Layout.contentHorizontalSpacing)
+                    .hidden() // Small hack to make the icon space consistent while not showing the icon.
+            }
+
+            VStack(alignment: .leading) {
+                Text(title)
+                    .bodyStyle()
+                    .padding(.top, Layout.contentVerticalSpacing)
+                Text(subtitle)
+                    .subheadlineStyle()
```

Comment:
It looks to me from the design that the font here should be `caption`.

Reply:
Good eye. Added in https://github.com/woocommerce/woocommerce-ios/pull/11634/commits/fcf0c2fbc7dc7f62dcb291a844e7e573a07128aa